### PR TITLE
Tidy up Relational Store

### PR DESCRIPTION
**Pull request by gs-kotang**
#### What type of PR is this?

- Improvement

#### What does this PR do / why is it needed ?

Minor changes in the relational store

- Remove code duplication
- Break up Tds window tests in SybaseIQ to make sure the integration tests run

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?
No

### CODE REVIEW

File: legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-execution/pom.xml [hunk 1]
```diff
@@ -188,6 +188,11 @@
             <artifactId>legend-engine-pure-code-compiled-core</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-relationalStore-executionPlan</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
```
Comment (rafaelbey) [inline legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-execution/pom.xml:195]: Can we confirm this setup wont end running test cases defined in this module twice?

Comment (gs-kotang) [reply legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-snowflake/legend-engine-xt-relationalStore-snowflake-execution/pom.xml:195]: They run only once. We need it to reuse AbstractTestSemiStructured without having to duplicate it in Snowflake module (in general each db extension that supports semi-structured data)

File: legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/test/semiStructured/AbstractTestSemiStructured.java [hunk 2]
```diff
@@ -133,4 +133,33 @@ private String readModelContentFromResource(String resourcePath)
...
+                TDSType +
+                "  (\n" +
+                "    SQL\n" +
+                "    (\n" +
+                "      type = Void\n" +
+                "      resultColumns = []\n" +
+                "      sql = ALTER SESSION SET QUERY_TAG = '{\"executionTraceID\" : \"${execID}\", \"engineUser\" : \"${userId}\", \"referer\" : \"${referer}\"}';\n" +
+                "      connection = RelationalDatabaseConnection(type = \"Snowflake\")\n" +
+                "    )\n" +
+                expectedRelational +
```
[the remark this answers is not in the document]
Comment (gs-kotang) [reply legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/test/semiStructured/AbstractTestSemiStructured.java]: Addressed this

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**